docs/section10/10-15.md

```diff
@@ -0,0 +1,58 @@
+- Eliminate duplicates in the list of zones in the proposed with humidification: `zones_have_humidification_list_p = list(set(zones_have_humidification_list_p))`  
+If humidification was modeled in the proposed for a specific zone, check if it was modeled in the baseline
+- For each zone_p in the P_RMR: `For zone_p in P_RMR...Zone:`
+    - Reset has_humidification_p in P_RMR boolean variable: `has_humidification_p = FALSE`
+    - Reset has_humidification_b in B_RMR boolean variable: `has_humidification_b = FALSE`
+    - Check if zone has humidification in the P_RMR, if it does then set boolean variable to true: `if zone_p in zones_have_humidification_list_p: has_humidification_p = TRUE`  
+    Check if zone was modeled with humidification in the B_RMR 
+    - Get analogous zone in the B_RMR: `zone_b = match_data_element(B_RMR,Zones,zone_p.id)`
+    - Get list of hvac systems that serve this zone in the B_RMR, should only be one, note that this RDS assumes only one: `hvac_list_b = get_list_hvac_systems_associated_with_zone(B_RMR,zone_b.id)`
+    - Check if the humidification_type does not equal Null or None, if it does then set has_humidification_b to True: `if hvac_b.humidification_type != Null and hvac_b.humidification_type != "None": has_humidification_b = TRUE`
+    
+    **Rule Assertion:**
+    - Case 1: If the zone was modeled with humidification in the baseline and proposed then Pass: `if has_humidification_b == TRUE and has_humidification_p == TRUE: outcome == "Pass"`
+    - Case 2: Else if baseline zone is modeled with humidification when the proposed zone does not have humidification: `elif has_humidification_b == TRUE and has_humidification_p == FALSE: outcome == "FAIL" and raise_message "The baseline was modeled with humidification when humidification was not modeled in the proposed design model".`  
+    - Case 3: Else if the baseline zone is not modeled with humidification when the proposed zone has humidification: `elif has_humidification_b == FALSE and has_humidification_p == TRUE: outcome == "FAIL" and raise_message "The baseline was NOT modeled with humidification when humidification was modeled in the proposed design model"`  
+    - Case 4: Else, humidification not modeled for the zone in proposed or baseline: `Else: outcome == "NOT_APPLICABLE"`  
+    
+**Notes**
+1. Based on the wording in Table G3.1 #10 baseline column, it means that if there was a scenario where one zone on a floor was served by a single zone system with humidification in the proposed, and a single system was required to be modeled per floor in the baseline that the whole system would have humidification. Not sure how many software tools would allow you to model humidification for just one zone. Maybe the assumption is that in this case G3.1.1 b would be triggered. The reason I thought about this is that maybe it is odd to be reporting this on the zone level.
```

Comment:
I think zone level is appropriate as there is too much variation between system coverage between proposed and baseline.

Any software tools that allow humidification in just one zone would probably do it through a second system or zone-level humidification, which means that you might need to check multiple HVAC systems in the baseline.  This is one of those situations where complying with one requirement (model humidification in the same zones as proposed) directly conflicts with another baseline requirement (one system per floor, and only one terminal per zone).  Though I think those cases will be rare.

Reply:
Agreed. I think this needs to be addressed in App G.